akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -370,7 +370,9 @@ object Flow {
 
 }
 
-/** Create a `Flow` which can process elements of type `T`. */
+/**
+ * A `Flow` is a set of stream processing steps that has one open input and one open output.
```

Comment:
Hmm, I'm not sure "has one open input and one open output" is quite true, a flow can complete its out and still consume its input etc. it also exists before materialized at all. I'd prefer something closer to the original/Scala doc (but without the "create" part like you mention)

Reply:
I copied the original Scala doc _verbatim_:

https://github.com/akka/akka/blob/master/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala#L38

I think "open" is used here in the conceptual sense, as in: a Flow has both a head and a tail that you can connect other components to.

Reply:
Ah, alright, I missed that it was a copy of existing (not great docs). That's fine then I guess.